package-utils.js

```diff
@@ -0,0 +1,81 @@
+var fs = require('fs');
+var path = require('path');
+var shell = require('shelljs');
+var ncp = require('child_process');
+
+//------------------------------------------------------------------------------
+// shell functions
+//------------------------------------------------------------------------------
+var rp = function (relPath) {
+    return path.join(shell.pwd() + '', relPath);
+}
+exports.rp = rp;

```

Comment:
Yes I have taken this functions from vsts-task repo. will add comment.
I am exporting it, because in future we may need this functions in other modules